src/Exception/NotImplemented.php

```diff
@@ -5,18 +5,12 @@
 namespace DBrekelmans\BrowserDriverInstaller\Exception;
 
 use LogicException;
-
 use function Safe\sprintf;
 
 /** @internal */
 final class NotImplemented extends LogicException
 {
-    private function __construct(string $message)
```

Comment:
The constructor needs to stay. It looks like it's not doing anything, but the `private` keyword is important here. It prevents you from calling `new NotImplemented()` and instead forces you to use the factory method `NotImplemented::feature()`.

Same in the other exception classes

Reply:
In PHP 7.1 we can not have a class extending `Exception` with private constructor. It is possible only since PHP 7.2
I know it sucks :-(
That's why I remove then these constructors as it makes no sense then to put them as public

Reply:
Ah I didn't know this was a php 7.2 feature. Good to know 👍